src/contracts/misc/GhoSteward.sol

```diff
@@ -0,0 +1,171 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.10;
+
+import {IPoolAddressesProvider} from '@aave/core-v3/contracts/interfaces/IPoolAddressesProvider.sol';
+import {IPoolConfigurator} from '@aave/core-v3/contracts/interfaces/IPoolConfigurator.sol';
+import {IPool} from '@aave/core-v3/contracts/interfaces/IPool.sol';
+import {DataTypes} from '@aave/core-v3/contracts/protocol/libraries/types/DataTypes.sol';
+import {PercentageMath} from '@aave/core-v3/contracts/protocol/libraries/math/PercentageMath.sol';
+import {GhoInterestRateStrategy} from '../facilitators/aave/interestStrategy/GhoInterestRateStrategy.sol';
+import {IGhoToken} from '../gho/interfaces/IGhoToken.sol';
+import {IGhoSteward} from './interfaces/IGhoSteward.sol';
+
+/**
+ * @title GhoSteward
+ * @author Aave
+ * @notice Helper contract for managing risk parameters of the GHO reserve within the Aave Facilitator
+ * @dev This contract must be granted `PoolAdmin` in the Aave V3 Ethereum Pool and `BucketManager` in GHO Token
+ * @dev Only the Risk Council is able to action contract's functions.
+ * @dev Only the Aave DAO is able to extend the steward's lifespan.
+ */
+contract GhoSteward is IGhoSteward {
+  using PercentageMath for uint256;
+
+  /// @inheritdoc IGhoSteward
+  address public constant AAVE_SHORT_EXECUTOR = 0xEE56e2B3D491590B5b31738cC34d5232F378a8D5;
+
+  /// @inheritdoc IGhoSteward
+  uint256 public constant MINIMUM_DELAY = 5 days;
+
+  /// @inheritdoc IGhoSteward
+  uint256 public constant BORROW_RATE_CHANGE_MAX = 0.0050e4;
+
+  /// @inheritdoc IGhoSteward
+  uint40 public constant STEWARD_LIFESPAN = 60 days;
+
+  /// @inheritdoc IGhoSteward
+  address public immutable POOL_ADDRESSES_PROVIDER;
```

Comment:
Minor, but not so much point of setting this as immutable and not the Pool and PoolConfigurator I think